### PR TITLE
added game loop

### PR DESCRIPTION
**Pull request by professionalSkillIssue**
added the loop to the game.

### CODE REVIEW

File: game.py [hunk 1]
```diff
@@ -1 +1,5 @@
-print("Welcome to our rock paper scissors game!")
\ No newline at end of file
+
+print("Welcome to our rock paper scissors game!")
+while True:
+    playerchoice = input("Please enter your choice from rock, paper or scissors. ")
```
Comment (RhysDaBoi) [inline game.py]: Could you make the variable name `player_choice`

**Review by RhysDaBoi — CHANGES_REQUESTED** [review]
Hey cool but could you change this issue

**Review by Mega-Roman — APPROVED** [review]
Very good, much better now :)

**Review by RhysDaBoi — APPROVED** [review]
Looks good

**Pull request merged**